be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -60,9 +68,16 @@ OLAPStatus AlphaRowset::copy(RowsetWriter* dest_rowset_writer) {
 }
 
 OLAPStatus AlphaRowset::remove() {
-    // TODO(hkp) : add delete code
-    // delete rowset from meta
-    // delete segment groups 
+    OlapMeta* meta = _data_dir->get_meta();
+    OLAPStatus status = RowsetMetaManager::remove(meta, rowset_id());
```

Comment:
If process core dump after remove rowset_meta, how to remove_files in segment_group?

Reply:
file gc